notification/job.go

```diff
@@ -292,6 +292,7 @@ func ProcessFallbackNotifications(parentCtx context.Context) (bool, error) {
 		var pending []models.NotificationSendHistory
 		if err := ctx.DB().Clauses(clause.Locking{Strength: clause.LockingStrengthUpdate, Options: clause.LockingOptionsSkipLocked}).
 			Where("status = ?", models.NotificationStatusAttemptingFallback).
+			Where("retries <= ?", ctx.Properties().Int("notification.max-retries", 4)).
```

Comment:
when the fallback is retried for more than `notification.max-retries`, it's status is changed to "error". So we weren't retrying fallbacks for more than `notification.max-retries` ?